packages/metro-resolver/src/types.js

```diff
@@ -78,6 +78,7 @@ export type PackageJson = $ReadOnly<{
   name?: string,
   main?: string,
   exports?: ExportsField,
+  imports?: ExportMap,
   ...
```

Comment:
This type name seem a bit odd now 🤔 Does that need an update?

Reply:
Yeah, while I was writing this it felt odd. While @huntie  originally suggested this, I could also see how renaming the type `ExportMap` to something like `PackageSubpathsMap` as a name that describes both import and export subpaths equally.